msteams-platform/toolkit/teams-toolkit-overview-visual-studio.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Teams Toolkit overview for Visual Studio
+author: surbhigupta
+description: In this module, learn Teams Toolkit Overview for Visual Studio
+ms.author: v-amprasad
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/24/2022
+---
+
+# Teams Toolkit overview for Visual Studio
+
+Teams Toolkit for Visual Studio helps you to create, debug and deploy Microsoft Teams apps. App development with Teams Toolkit has the advantages of:
```

Comment:
Please try to mention in begining that Teams Toolkit for Visual Studio is GA in Visual Studio 2022 v17.3. Let customer know all this is based on VS 2022 v17.3. In other versions they will  not see Teams Toolkit.

Reply:
This comment has been implemented.